okhttp-hpacktests/src/test/java/okhttp3/internal/http2/HpackDecodeTestBase.java

```diff
@@ -41,7 +43,7 @@
     for (String interopTestName : interopTests) {
       List<Story> stories = HpackJsonUtil.readStories(interopTestName);
       if (stories.isEmpty()) {
-        fail("No stories for: " + interopTestName);
+        throw new AssumptionViolatedException("No stories for: " + interopTestName);
```

Comment:
The past-tense of this class bothers me. Shouldn't it be `AssumptionViolationException`?

Reply:
It seems like the dynamic tests cases don't work with assumption failures, so I've replaced with just not running any tests if that is the case.

Reply:
Too much Kotlin.  apologies for the shoddy diffs recently.